cli/src/target.ts

```diff
@@ -62,7 +66,10 @@ export default class Target {
 
   async clean() {
     // Remove the directory associated with this target.
-    await remove(path.resolve(this.crate.root, 'target', this.subdirectory));
+    const absolutePathSubdir = path.resolve(this.crate.root, 'target', this.subdirectory);
+    if (existsSync(absolutePathSubdir)) {
```

Comment:
You don't need the `existsSync` check here; I'm pretty sure `rimraf` will silently succeed even if the directory already doesn't exist.